packages/odata-service-inquirer/src/prompts/edmx/questions.ts

```diff
@@ -54,7 +54,7 @@ function validateEntityChoices(
 
     if (!PromptState.isYUI && validationMsg) {
         LoggerHelper.logger.debug(`Exiting due to validation error: ${validationMsg}`);
-        throw new Error(t('errors.exitingGeneration', { exitReason: validationMsg }));
+        throw new Error(`${t('errors.exitingGeneration')} exitReason: ${validationMsg}`);
```

Comment:
This is a technical error, Im not sure why its being changed as it may break error handling.